### DISCUSSION TITLE
How does UniformTSDFVolume.integrate & ScalableTSDFVolume.integrate work ?

### DISCUSSION THREAD

**Discussion by CyrielleAlbert** [isl-org/Open3D#4311] [Q&A]
I am using the example [here](https://github.com/isl-org/Open3D/blob/master/examples/python/reconstruction_system/make_fragments.py) with my own dataset of 15 RGBD images. I managed to visualize each image by building point_cloud of each so my images seem correct.
However, when I run the make_fragment program, integrate_rgb_frames_for_fragment returns a mesh with 0 points. I checked the different steps of the algorithm and it seems like volume.integrate() does not work as I would have wanted to (the volume always has 0 points) . But no error thrown.
Do you have any idea of what could be the issue? 
Could it be because the images are too different or would it still have returned something?  
Any idea of where is implemented that function in the code so that I could take a look at it?

Here are some screenshots.
![image](https://user-images.githubusercontent.com/43926352/142380208-0e83ef71-20e4-4534-9105-350d15c508c6.png)
![image](https://user-images.githubusercontent.com/43926352/142380326-a3d59968-ceaf-4077-81a7-40c89bd82cdb.png)

PS:  I have built my depth images using the MiDaS algorithm (and inversing the values) and I am aware that the depth images are not in meters and not all the same scale.

**Comment by germanros1987** [comment]
Could you please provide the fragments?
